perf/perf_nmem.py

```diff
@@ -229,8 +231,9 @@ def test_cpumask_cpu_off(self):
             self.fail("Can't offline cpumask cpu %s" % disable_cpu)
         current_cpu = self._get_cpumask(pmu1)
         self.log.info("Current CPU: %s" % current_cpu)
-        self._check_cpumask()
         # After confirming cpu got disabled, enable back
         if current_cpu in online_cpus and disable_cpu != current_cpu:
             if cpu.online(disable_cpu):
+                self.log.info("cpu enabled back %s" % online_cpus)
+            else:
```

Comment:
please push numa changes and loging  in to another commit

Reply:
sent separate commit